content/en/seachat/seachat-manual/03-add-knowledge/05-document-upload.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Upload Documents"
+description: "Upload More Than 20 Types of Documents to SeaChat Agent"
+lead: ""
+date: 2024-04-11 10:43:51.069 +0100
+lastmod: 2024-04-11 10:43:51.069 +0100
+weight: 33
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/03-add-knowledge/template-upload/
+---
+
+## Overview
+SeaChat provides four methods to upload files to your agent. We will focus on the **Upload Documents** method in this tutorial and by the end of the tutorial your SeaChat agent will have a customized knowledge base at your service.
+
+
+## Create a SeaChat Agent
+If you don't have a SeaChat account yet, you can sign up for free at [SeaChat website](https://chat.seasalt.ai/)! You can find all the information you need to create a knowledge-based AI agent in [Create an Agent](/en/seachat/seachat-intro/02-how-to-create/).
+
+
+## Open Knowledge Base
+Find your agent's knowledge base by navigating to the **Knowledge Base** dashboard under **Agent Configuration** in the sidebar menu. Choose **Upload Documents** and have the more than 20 types of documents ready to be uploaded to your agent.
+
+<br/>
+<center>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-add-knowledge/document-upload/choose-document-upload.png" alt="Image of the Knowledge Base dashboard through the Agent Configuration in the sidebar menu to show how to upload a CSV or JSON file to an agent by selecting Upload from Template File.">
+
+*Knowledge Base Dashboard*
+</center>
+
+## Types of Documents
+SeaChat supports more than 20 types[^1] of documents that you can upload to your agent. In this Tutorial, we will show you how to upload a whole webpage as a document to your agent. 
+
+Simply download the website that you want your AI agent to learn as .html file. In chrome, you can save a webpage as .html by right-clicking on the webpage and selecting **Save as**.
+
+> :rotating_light: **Note** :rotating_light:
+>
+> For audio files e.g. mp3 and picture files e.g. png, we only support English documents.
```

Comment:
Mention that we also have audio and video upload.

> For audio files e.g. mp3 we only support English documents.

this is kinda outdated because we later added the multimedia upload function. So you can upload Chinese/English audio there

Suggestion: For picture files e.g. png, we only support English documents. For best result on audio files, we suggest you use audio/video upload(add link)